pkg/packagekit/package_wix.go

```diff
@@ -173,6 +175,34 @@ func PackageWixMSI(ctx context.Context, w io.Writer, po *PackageOptions, include
 	return nil
 }
 
+// getSigntoolPath attempts to look up the location of signtool so that
+// we do not have to rely on a hard-coded signtool location that will change
+// when we upgrade to a new version of signtool.
+func getSigntoolPath() string {
+	var signtoolPath string
+	root := `C:\Program Files (x86)\Windows Kits` // restrict our lookup to a well-known location
+	fileSystem := os.DirFS(root)
+
+	if err := fs.WalkDir(fileSystem, ".", func(currentPath string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		if !d.IsDir() && strings.HasSuffix(currentPath, `x64/signtool.exe`) {
+			signtoolPath = filepath.Join(root, currentPath)
+		}
+		return nil
+	}); err != nil {
+		return defaultSigntoolPath
+	}
+
+	if signtoolPath == "" {
+		// Not found -- default to well-known version/location
+		return defaultSigntoolPath
+	}
```

Comment:
I think we should error out, instead of hard coding a default. After all, if we don't find that walking, something is real fishy....

Reply:
That's reasonable! I will update.